lib/logs.go

```diff
@@ -46,9 +46,6 @@ func (c *ContainerServer) GetLogs(container string, logChan chan string, opts Lo
 			continue
 		}
 		logMessage := line.Text[secondSpaceIndex(line.Text):]
-		if opts.Details {
-			// add additional information to line
```

Comment:
Do we want to file a placeholder issue for this?

Reply:
https://github.com/kubernetes-incubator/cri-o/issues/1547